### ISSUE TITLE
Anti-21 hydroxylase

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7239]
**Preferred term label:**
Anti-21-hydroxylase antibody positivity

**Synonyms**
P450c21

**Definition (free text, please give PubMed ID)**
PMID: 22707647, PMID: 9030872

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:85138

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
added
